scripts/tests/test_check_markdown_format.py

```
@@ -0,0 +1,259 @@
+"""Exercise the Markdown formatter checker at its process boundary."""
+
+import json
+import os
+import subprocess  # ruff: ignore[suspicious-subprocess-import] - the boundary is under test.
```

Ruff (0.16.3)

[warning] Invalid rule code in suppression: suspicious-subprocess-import

Enable `lint.preview` to use rule names

(RUF102)